pkg/sql/contention/txnidcache/concurrent_write_buffer.go

```diff
@@ -20,6 +20,10 @@ const messageBlockSize = 1024
 
 type messageBlock [messageBlockSize]ResolvedTxnID
```

Comment:
If I were to be nit-picky, I like I'd just call this `block`. The fact that things are "messages" sort of muddles things for me. I think I'd go through and eliminate the word `message`/`msg` everywhere. `messageSink` would become `blockSink`, `messageBlockSize`->`blockSize` `msgBlockPool`->`blockPool`. Then for the list, maybe `blockListNode` and `blockList`?